Zhukov/t9.py

```diff
@@ -0,0 +1,22 @@
+
+# Напишите функцию Python, чтобы проверить, является ли строка панграммой
+# Примечание. Панограммы — это слова или предложения, содержащие каждую букву алфавита хотя бы один раз.
+# Например: «Быстрая коричневая лиса перепрыгивает через ленивую собаку»
+#
+# Экс-граф? Плюш изъят. Бьём чуждый цен хвощ!
+# Любя, съешь щипцы, — вздохнёт мэр, — кайф жгуч.
+
+def is_pangram(string):
+    return set('абвгдеёжзийклмнопрстуфхцчшщъыьэюя') - set(string.lower())
+    
+
+def main():
+    print('Введите строку: ','\n')
+    s = input()
+    if(is_pangram(s) == True):
```

Comment:
```suggestion
    if is_pangram(s):
```
Сравнение с bool выражения, результатом которого и так является bool, является плохой практикой.
применить везде